keps/sig-storage/3294-provision-volumes-from-cross-namespace-snapshots/kep.yaml

```diff
@@ -0,0 +1,56 @@
+title: Provision volumes from cross-namespace snapshots
+kep-number: 3294
+authors:
+  - "@Elbehery"
+  - "@mkimuram"
+owning-sig: sig-storage
+participating-sigs:
+  - sig-storage
```

Comment:
sig-storage can be removed here.  This is for additional sigs.  Since we are using an Alpha CRD from sig-networking, it may be good to add sig-networking here.